src/test/java/com.example.online_shop_api/PasswordServiceTest.java

```diff
@@ -0,0 +1,160 @@
+package com.example.online_shop_api;
```

Comment:
It is good that there is unit test for the service class. Would be even better to have a couple of integration tests of the controller - to test with database.
But this can be a separate task because you still don't have test DB in the project.